target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -385,6 +415,8 @@
 			fixed-link {
 				speed = <1000>;
 				full-duplex;
+				pause;
+                asym-pause;
```

Comment:
indent/whitespace issue

Reply:
should go to separate commit `Set asym-pause and pause support for gmac driver.`